ext/storage_sources/azure_store/README.md

```diff
@@ -0,0 +1,84 @@
+# WiredTiger's Azure Extension
+## 1. Introduction
+This extension allows WiredTiger storage source extensions to read from and write to objects stored
+in Azure Blob Storage using WiredTiger’s provided internal abstraction for storing data in an object
+storage service.
+
+## 2. Building and running
+This section describes how to build WiredTiger with the Azure extension enabled.
+
+### Requirements
+* CMake 3.13 or higher
+* G++ 8.4 or higher
+* clang version 6.0.0
```

Comment:
I think we should also say or higher, to denote that its forward compatible as wlel. 
```suggestion
* clang version 6.0.0 or higher
```

Reply:
addressed https://github.com/wiredtiger/wiredtiger/pull/8802/commits/1329d79760b71f8779343c5eee6b23f5933dced0